### ISSUE TITLE
Concept events also trigger as an asset

### ISSUE THREAD

**Issue by EmberLightVFX** [cgwire/zou#775]
**Context**

Studio name: Ember Light
Zou version: 0.18.7
Zou installation type: docker

**Describe the bug**
Running gazu event listeners on both assets and concepts fires both events when I create a new concept.
The assets ID is the same as the concepts ID when checking the event-data:

```
{'concept_id': '7ea38937-e5f5-469e-8c23-6623e0ff8f94', 'project_id': '67c8ad65-0350-4b38-93be-5d0a4653f465'}
{'asset_id': '7ea38937-e5f5-469e-8c23-6623e0ff8f94', 'project_id': '67c8ad65-0350-4b38-93be-5d0a4653f465'}
```

**Expected behavior**
Concepts should only trigger concept-events

**Comment by EvanBldy** [comment]
Hi @EmberLightVFX,
Thanks for the report ! 
It should be fixed in the last version of Zou (0.19.0). 
Can you close this issue if it's ok for you ?

**Comment by EmberLightVFX** [comment]
Awesome! Thanks you :)
